tests/testthat/test-carehomes-check.R

```diff
@@ -48,6 +48,38 @@ test_that("everyone is infected when beta is large", {
 })
 
 
```

Comment:
Is it worth adding a test that if waning rate is zero then noone ever goes back? You might need to turn off some other processes to make this really obvious, and it might not be worthwhile. We do have a test of this already to a degree in that the reference example tests are passing (so you do have the same model!)

Reply:
@edknock has added something like this [here](https://github.com/mrc-ide/sircovid/blob/802a521995d03910d2fac191d40b7e32ec48a8dd/tests/testthat/test-carehomes-check.R#L84)